tools/ci_build/github/linux/docker/scripts/install_deps.sh

```diff
@@ -113,8 +113,14 @@ if [ $DEVICE_TYPE = "Normal" ]; then
     ${PYTHON_EXE} -m pip install sympy==1.1.1
 elif [ $DEVICE_TYPE = "gpu" ]; then
     ${PYTHON_EXE} -m pip install sympy==1.1.1
+    # Inferencing pipelines need Torch installed as well to test IOBinding
+    if [ "$PYTHON_VER" = "3.5" ]; then
+        echo "Python 3.6 and above is needed for installing the dev Torch package!"
+    else
+        echo "Installing the dev Torch package"
+        ${PYTHON_EXE} -m pip install --upgrade --pre torch==1.6.0.dev20200610 torchvision==0.7.0.dev20200610 -f https://download.pytorch.org/whl/nightly/cu101/torch_nightly.html
```

Comment:
Then you must register these things in cgmanifest.json.

Reply:
Wasn't torch already being installed as part of the training pipeline even before this change? I would assume the training team already took care of updating cgmanifest, etc. I see this in the cgmanifest.json file today.

      {
         "component": {
            "type": "git",
            "git": {
               "commitHash": "90537289a04ef5d572496240e2ac3a881be518d2",
               "repositoryUrl": "https://github.com/pytorch/pytorch.git"
            }
         }
      },

Reply:
@pranavsharma It's not enough. 
1. Does the version match?
2. Where are the dependencies?